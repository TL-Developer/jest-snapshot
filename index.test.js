'use strict';

const responseServer = () => [
  'js',
  'java',
  'c#',
];

it('Test Snapshot', () => {
  expect(responseServer()).toMatchSnapshot();
});
