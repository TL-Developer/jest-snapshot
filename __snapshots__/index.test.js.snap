// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Snapshot 1`] = `
Array [
  "js",
  "java",
  "c#",
]
`;
